pyspecdata/core.py

```diff
@@ -308,8 +308,24 @@ def mybasicfunction(first_figure=None):
 
 
 def issympy(x):
+<<<<<<< HEAD
+    'tests if something is sympy (based on the module name)'
+    return isinstance(x,sympy_expr)
+def psd_sqrt(arg):
+    "pyspecdata square root -- this is the easiest way to get good error propagation"
+    if isinstance(arg,nddata):
+        return arg**0.5
+    elif isinstance(arg,sympy_symbol):
+        return sympy_sqrt(arg)
+    else:
+        return np_sqrt(arg)
+||||||| d1d83c22
+    'tests if something is sympy (based on the module name)'
+    return isinstance(x,sympy_expr)
+=======
     "tests if something is sympy (based on the module name)"
     return isinstance(x, sympy_expr)
+>>>>>>> master
```

Comment:
this conflict needs resolution -- you just left the conflict markers in.

Reply:
the middle block is the starting point, and the thing on top vs. thing on bottom are the two conflicting changes to the middle block.  Typically, you want to do both.

e.g. here, the top block added the psd_sqrt function, while the bottom block black-formatted the strings.  So, you probably want something like the top block, but where the strings are black formatted.  Be sure to remove the conflict markers!

Reply:
yeah, actually just give me a minute